packages/contracts/deploy/helpers.ts

```diff
@@ -129,7 +130,7 @@ export function getLatestContractAddress(
 
   const osxNetworkName = getNetworkNameByAlias(networkName);
   if (!osxNetworkName) {
-    if (networkName === 'hardhat') {
+    if (networkName === 'hardhat' || networkName === 'localhost') {
```

Comment:
```suggestion
  if (
    networkName === 'hardhat' ||
    networkName === 'coverage' ||
    networkName === 'localhost'
  ) {
```
In the past, the network name 'coverage' was possible too. Not sure if this is outdated by now.

Reply:
Thanks I'll check